homeassistant/config_entries.py

```diff
@@ -706,3 +715,16 @@ def __init__(self, hass: HomeAssistant) -> None:
 
         result["result"] = True
         return result
+
+
+class SystemOptions:
+    """"""
+
+    def __init__(self, data: dict) -> None:
+        """"""
+        self.data = data
+
+    @callback
+    def update(self, disable_new_entities: bool) -> None:
```

Comment:
Let's force keywords only

```suggestion
    def update(self, *, disable_new_entities: bool) -> None:
```